waspc/examples/todo-typescript/.wasp/out/sdk/wasp/package.json

```diff
@@ -21,12 +21,15 @@
       "./api": "./dist/api/index.js",
       "./api/*": "./dist/api/*",
       "./operations": "./dist/operations/index.js",
+      "./ext-src/*": "./dist/ext-src/*.js",
```

Comment:
Not ideal, but necessary. See [this comment](https://github.com/wasp-lang/wasp/pull/1626/commits/0c6ee6c9ffd839a7be6fd2d509599fc77fecf333#r1459095058).